lib/suse/connect/connection.rb

```diff
@@ -66,6 +66,9 @@ def json_request(method, path, params = {})
           body: body,
           success: response.is_a?(Net::HTTPSuccess)
         )
+
+      rescue Zlib::Error
+        raise SUSE::Connect::NetworkError, 'Network error. Try again and report a bug if it keeps happening.'
```

Comment:
I don't think so. According to [this comment](https://bugzilla.suse.com/show_bug.cgi?id=982630#c3) all what is asked for is a generic _"please try again"_ message. Other messages like this in YaST or Connect have no bugzilla link either.